sdk/core/azure-core-test/src/test/java/com/azure/core/test/TestProxyTests.java

```diff
@@ -290,6 +290,31 @@ public void testBodyRegexRedactRecord() {
         assertEquals(record.getResponse().get("TableName"), REDACTED);
     }
 
+    @Test
+    @Tag("Playback")
+    public void testRedactRequestBodyRegex() {
+
+        HttpClient client = interceptorManager.getPlaybackClient();
+        HttpPipeline pipeline = new HttpPipelineBuilder().httpClient(client).build();
+
+        //        HttpClient client = new HttpURLConnectionHttpClient();
+        //        HttpPipeline pipeline = new HttpPipelineBuilder().httpClient(client).policies(interceptorManager.getRecordPolicy()).build();
```

Comment:
I left it because for our infra tests here, when we change something for a playback we re-record it with the same method. So this is the "record" path that isn't persisted but handy to have around.